TFT/src/User/SanityCheck.h

```diff
@@ -25,5 +25,8 @@
   #define STARTUP_KNOB_LED_COLOR 1
 #endif
 
+#ifdef ST7920_FULLSCREEN
+  #error "Fullscreen Marlin Mode is enabled by default and ST7920_FULLSCREEN is now ST7920_WINDOWED. Please update your configuration."
+#endif
```

Comment:
```suggestion
#if defined(TFT24_V1_1) && !defined(ST7920_WINDOWED) 
  #error "Fullscreen Marlin Mode is not recommended for TFT24 Variants. Please Enable ST7920_WINDOWED in Configuration.h."
#endif
#ifdef ST7920_FULLSCREEN
  #error "Fullscreen Marlin Mode is enabled by default and ST7920_FULLSCREEN is now ST7920_WINDOWED. Please update your configuration."
#endif
```

Reply:
`TFT24_V1_1` doesn't every get defined before the sanity check, so this won't work.

Reply:
This just checks for an old config setting.

Reply:
I just tried to compile with the non-recommended and it works.